src/core/event/EventManager.js

```diff
@@ -0,0 +1,60 @@
+/* global console */
+
+export default class EventManager {
+  constructor(nextHandlers) {
+    this._nextHandlers = nextHandlers;
+  }
+
+  attach(element) {
+    for (const eventType of this.constructor.handledRawEventTypes) {
+      element.addEventListener(eventType, this[eventType]);
+    }
+  }
+
+  _mergeHandlers(eventType, ...handlers) {
+    return function handleEventMerged(...args) {
+      for (const handler of handlers) {
+        try {
+          handler(...args);
+        } catch (err) {
+          // eslint-disable-next-line no-console
+          console.error(
+            `Exception occured in member of merged event handler "${eventType}": `,
+            err
+          );
+        }
+      }
+    };
+  }
+
+  getMiddlewareHandlers() {
```

Comment:
The naming here:
- What do we mean with 'middleware' in this context?
- No underscore - is this a public function? If so when should it be called?
- luma classes usually have public methods first, followed by private (underscore) methods.

Again JSDoc would come to the rescue here.